src/transformers/models/convnext/configuration_convnext.py

```diff
@@ -67,11 +67,11 @@ class ConvNextConfig(PretrainedConfig):
 
     Example:
     ```python
-    >>> from transformers import ConvNextModel, ConvNextConfig
+    >>> from transformers import ConvNextConfig, ConvNextModel
 
     >>> # Initializing a ConvNext convnext-tiny-224 style configuration
     >>> configuration = ConvNextConfig()
-    >>> # Initializing a model from the convnext-tiny-224 style configuration
+    >>> # Initializing a model (with random weights) from the convnext-tiny-224 style configuration
```

Comment:
Let's add an empty line before this line

Reply:
Also an empty line between `>>> model = ConvNextModel(configuration)` and `>>> # Accessing the model configuration`